### DISCUSSION TITLE
How to get Achievements badge in 2022?

### DISCUSSION THREAD

**Discussion by proarafat** [community/community#23101] [New to GitHub]
<p>how to get the Achievements badge in 2022? or is there any chance to get this Achievements badge in 2022? and how ???</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there <a class="mention" href="/proarafat">@proarafat</a> :wave: When you participate in certain programs, GitHub will display the badge. To learn more about the programs and criteria to have badges displayed, check out our documentation <a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile">here</a>.</p>

**Reply by elishakuje** [reply]
This was very helpful.

**Reply by rishunayak** [reply]
yes thanks

**Reply by sabrinachowdhuryoshin** [reply]
Helpful. Thanks!

**Reply by DilanRamirez** [reply]
thank you

**Comment by proarafat** [comment]
<p>thank you <a class="mention" href="/liligalante">@liligalante</a> :slightly_smiling_face:</p>

**Reply by rishunayak** [reply]
yup

**Comment by Polobear9** [comment]
<p>you can check the badge in here<br>
—&gt; URL : <a href="https://github.com/Schweinepriester/github-profile-achievements" class="inline-onebox" rel="noopener nofollow ugc">GitHub - Schweinepriester/github-profile-achievements: A collection listing all Achievements available on the GitHub profile 🏆</a></p>
<p>:wink:</p>

**Reply by Kodiererin** [reply]
Thanks Man!

**Comment by iamKonick** [comment]
Hi, The achievement badges are a kind of reward as you learn and become better as a developer and using GitHub. YoLo, Shark, Pull these badges are easy to get.

**Comment by tuhinmallick** [comment]
Hey! can you please mark my answer

**Reply by gosham2016** [reply]
Нет

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @proarafat. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
